frontend/src/components/Answer.jsx

```diff
@@ -1,16 +1,25 @@
 import PropTypes from "prop-types";
+import { useState } from "react";
 
 function Answer({ title, res }) {
-  return (
-    <button
-      type="submit"
-      className="answer"
-      onClick={
-        res
-          ? () => window.alert("Bonne réponse !")
-          : () => window.alert("Mauvaise réponse !")
+  const [isActive, setActive] = useState(false);
+
+  const ToggleClass = () => {
+    setActive(true);
+  };
+
+  function resultStyle() {
+    if (isActive) {
+      if (res === true) {
+        return "good-answer";
```

Comment:
a good answer is still an answer. you should return `"answer good-answer`. this will allow you to benefit from the answer class in the css